tests/utils/test_minio_crud_client.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import os
+import io
+import time
+
+from pathlib import Path
+
+project_root = str(Path(__file__).resolve().parent.parent.parent)
+sys.path.append(project_root)
```

Comment:
it's better to use PYTHONPATH env variable instead 
https://bic-berkeley.github.io/psych-214-fall-2016/using_pythonpath.html

Reply:
updated

Reply:
great